parts/chapters/subsections/5.3/CO2SOL.fodt

```diff
@@ -4203,20 +4202,20 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
     <text:p text:style-name="P52">The CO<text:span text:style-name="T20">2</text:span>-Brine PVT model computes the PVT properties such as density, viscosity, and enthalpy internally as functions of pressure, temperature, and composition by using analytic correlations and models from the literature rather than by interpolation <text:span text:style-name="T21">of</text:span> tabulated values <text:span text:style-name="T21">in the input deck</text:span>. These values are transformed to the standard black-oil equivalent PVT tables internally by the simulator. A full description of the underlying PVT models is described by Sandve et al.<text:note text:id="ftn1" text:note-class="footnote"><text:note-citation>1</text:note-citation><text:note-body>
-       <text:p text:style-name="Footnote">Tor Harald Sandve1, Sarah E. Gasda, Atgeirr Rasmussen, <text:span text:style-name="T22">and</text:span> Alf Birger Rustad. Convective dissolution in field scale CO<text:span text:style-name="T20">2</text:span> storage simulation using the OPM Flow simulator. Submitted to TCCS 11 – Trondheim Conference on CO<text:span text:style-name="T20">2</text:span> Capture, Transport and Storage Trondheim, Norway – June 21-23, 2021.</text:p></text:note-body></text:note>. Note that the CO<text:span text:style-name="T20">2</text:span>-Brine PVT properties depend on the temperature and salinity and these must therefore be entered in the PROPS section. The reservoir temperature can be defined using, e.g., the RTEMP keyword. Region based salinity can be provided using the SALINITY keyword.</text:p>
-    <text:p text:style-name="P52">The CO2SOL keyword must be used together with the SOLVENT keyword in the RUNSPEC section. The SOLVENT keyword activates the four component black-oil model: oil, water and gas, plus a solvent (in this case CO<text:span text:style-name="T20">2</text:span>).</text:p>
-    <text:p text:style-name="P52">The DISGASW keyword in the RUNSPEC section can be used to model dissolution of CO<text:span text:style-name="T20">2</text:span> in the Brine. <text:span text:style-name="T21">Note that hydrocarbon gas is not allowed to dissolve in the Brine (this is considered a reasonable assumption for most hydrocarbon gases).</text:span></text:p>
-    <text:p text:style-name="P52">The hydrocarbon <text:span text:style-name="T23">reservoir fluid</text:span> properties should be set up as normal apart from the water PVT properties, which will be defined by the simulators’ CO<text:span text:style-name="T20">2</text:span>-Brine PVT model, so the water PVT keywords (e.g. PVTW) are not required. The CO<text:span text:style-name="T20">2</text:span> PVT properties will also be defined by the CO<text:span text:style-name="T20">2</text:span>-Brine PVT model so the solvent PVT keywords (e.g. SDENSITY, PVTSOL) are also not required.</text:p>
```

Comment:
PVTSOL has not been converted to a link. This seems to be due to a typo in the file name PVSOL.fodt not PVTSOL.fodt.

Reply:
Thanks for spotting that, I have added #415 to fix the filename.